rules/azure/execution_command_virtual_machine.toml

```diff
@@ -46,7 +46,7 @@ event.module:azure and event.dataset:azure.activitylogs and event.category:Admin
 framework = "MITRE ATT&CK"
 [[rule.threat.technique]]
 id = "T1059"
-name = "Command-Line Interface"
+name = "Command and Scripting Interpreter"
```

Comment:
No new sub-techniques were used in the update, but existing name changes were updated